### PR TITLE
Jeos vm #8027 ke

### PR DESCRIPTION
**Pull request by keichwa**
close https://github.com/SUSE/spacewalk/issues/8027

### CODE REVIEW

File: modules/installation/pages/install-vm.adoc [hunk 1]
```diff
@@ -78,29 +78,65 @@ During the basic installation prompts you are asked to enter the root password.
...
+rpm -e --nodeps sles-release
+----
+
+. Register {productname} with SCC:
 +
 
 ----
-SUSEConnect -e<EMAIL_ADDRESS> -r<SUSE_MANAGER_CODE>
+SUSEConnect -e<EMAIL_ADDRESS> -r<SUSE_MANAGER_CODE> \
+  -p SUSE-Manager-Server/<productnumber>/x86_64
```
Comment (mcalmer) [inline modules/installation/pages/install-vm.adoc]: Server is available for 3 architectures.
When we work here with place holder we should do it also for the arch.

File: modules/installation/pages/install-vm.adoc [hunk 2]
```diff
@@ -78,29 +78,65 @@ During the basic installation prompts you are asked to enter the root password.
...
 ----
-SUSEConnect -p SUSE-Manager-Server/<productnumber>/x86_64 -r<SUSE_MANAGER_CODE>
+SUSEConnect -p sle-module-basesystem/15.1/x86_64
+SUSEConnect -p sle-module-python2/15.1/x86_64
+SUSEConnect -p sle-module-server-applications/15.1/x86_64
+SUSEConnect -p sle-module-web-scripting/15.1/x86_64
+SUSEConnect -p sle-module-suse-manager-server/<productnumber>/x86_64
+----
++
+TIP: JeOS is configured to install only Required packages. To get all
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]: Either remove the initcap, or mark up "required"

Comment (Loquacity) [reply modules/installation/pages/install-vm.adoc]: Also needs a newline for the new sentence.

File: modules/installation/pages/install-vm.adoc [hunk 3]
```diff
@@ -113,27 +149,13 @@ zypper in -t package yast2-storage
 | VirtIO Disk 4        | [path]``swap``           | 4{nbsp}GB    | swap
 |===
 
-. If you are still using an older version than {susemgr}{nbsp}3.2 check [path]``/etc/fstab`` for correctness as follows (_updated tools shipped with recent SPs will no longer require human intervention._):
-Remove or comment out this mount point entry for [path]``/var/lib/pgsql/`` in the [path]``/etc/fstab`` file:
+. Exit the partitioner and install the {productname} pattern (approx. 730 packages, 1.4 GB when installed):
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
. Exit the partitioner and install the {productname} pattern (approximately 730 packages, consuming 1.4 GB when installed):
```

File: modules/installation/pages/install-vm.adoc (hunk 1 above)
Comment (keichwa) [reply modules/installation/pages/install-vm.adoc]: ok, fair req

File: modules/installation/pages/install-vm.adoc (hunk 2 above)
Comment (keichwa) [reply modules/installation/pages/install-vm.adoc]: ok, prepare to push

**Comment by keichwa** [comment]
> Make sure you uncomment the PDF includes as well.

ah, good advise!

**Pull request merged**